app/actions/authInfo/validateRoles.js

```diff
@@ -1,31 +1,46 @@
 import axios from 'axios';
-import _ from 'lodash';
 import { axiosParameters } from '../../../lib/axios/signupActions';
 export const CHECK_ROLES_SUCCESS = 'CHECK_ROLES_SUCCESS';
 
-export const validateRolesSuccess = (authorized) => {
+const INVOICESIGNUPROLES = [
+  'signup_service_create_invoice_signup_staging',
+  'signup_service_create_invoice_signup_prod'
+];
+const TRUSTSIGNUPROLES = [
```

Comment:
To help readability....
```suggestion
const INVOICE_SIGNUP_ROLES = [
  'signup_service_create_invoice_signup_staging',
  'signup_service_create_invoice_signup_prod'
];
const TRUST_SIGNUP_ROLES = [
```